### PR TITLE
ticket 7 completed

### PR DESCRIPTION
**Pull request by RichardNeat**
ticket 7 - refactored GET reviewsById to include the comment_count - SQL query working correctly and test refactored, now passing.

### CODE REVIEW

File: models/reviews.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 const db = require('../db/connection');
 
 exports.selectReviewById = (review_id) => {
-    return db.query('SELECT * FROM reviews WHERE review_id = $1;', [review_id])
+    return db.query('SELECT reviews.*, COUNT(comments.body) AS comment_count FROM reviews JOIN comments ON comments.review_id = reviews.review_id WHERE reviews.review_id = 2 GROUP BY reviews.review_id;')
```
Comment (itskatherine) [inline models/reviews.js:4]: You're missing SQL injection 💉  here, so it'll only work for the review with ID 2

File: models/reviews.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
         .then((response) => {
+            console.log(response.rows);
```
Comment (itskatherine) [inline models/reviews.js:6]: I'd get rid of this console log now we don't need it 🙂

**Comment by itskatherine** [comment]
Great work, speeding along! Some changes needed (outlined above), whack another pull request through once you've addressed them 🔧

File: models/reviews.js (hunk 1 above)
Comment (RichardNeat) [reply models/reviews.js:4]: Good spot thank you!

File: models/reviews.js (hunk 2 above)
Comment (RichardNeat) [reply models/reviews.js:6]: Thank you will do :)

**Pull request merged**